src/common/ImageAvatar.js

```diff
@@ -25,17 +25,18 @@ export default class ImageAvatar extends PureComponent {
       width: size,
     };
 
-    const imageStyle = {
-      ...touchableStyle,
-      borderRadius:
-        shape === 'rounded' ? size / 8 : shape === 'circle' ? size / 2 : shape === 'square' ? 0 : 0,
-    };
+    const borderRadius =
+      shape === 'rounded' ? size / 8 : shape === 'circle' ? size / 2 : shape === 'square' ? 0 : 0;
 
     return (
       <Touchable onPress={onPress} style={touchableStyle}>
-        <Image style={imageStyle} source={{ uri: avatarUrl }} resizeMode="contain">
+        <ImageBackground
+          style={touchableStyle}
+          source={{ uri: avatarUrl }}
+          resizeMode="cover"
```

Comment:
made this change due to https://github.com/facebook/react-native/issues/8885#issuecomment-281023137